index.html

```diff
@@ -0,0 +1,128 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Home Page Capstone Project</title>
+    <link rel="stylesheet" href="style.css"/>
+    <link rel="stylesheet" href="https://use.fontawesome.com/releases/v5.14.0/css/all.css">
+ 
+</head>
+<body>
+    <header id="dsktop-header">
+        <nav id="nav1" title="first-navigation">
+          <ul>
+            <li><a class="fa-brands fa-facebook-f"></a></li>
+            <li><a class="fa-brands fa-twitter"></a></li>
+            <li><a href="#" class="nav1item">Swahili</a></li>
+            <li><a href="#" class="nav1item">My Page</a></li>
+            <li><a href="#" class="nav1item">Logout</a></li>
+          </ul>
+        </nav>
+         
+        <nav id="nav2" title="second-navigation"> 
+          <a class="main-logo" href="#home"><img src="./sources/logo.png" alt="logo"></a>
```

Comment:
- [ ] it could be better if you put your logo to the le as on the design. Kindly fix it 👍 

![Capture01](https://user-images.githubusercontent.com/57408419/225955524-8af72b80-848a-4894-8f9c-9e2a8a573d00.PNG)
![Capture02](https://user-images.githubusercontent.com/57408419/225955564-80ab157f-e5c3-4b7c-a6aa-4c5e18b379ba.PNG)

Reply:
Kindly move your logo to the left like on the design. On the design, there is an equal space around the header. On yours, there is more space on the left. Plese align the header to the center like on the design 👍 
<img width="1036" alt="Screenshot 2023-03-17 at 9 21 12 PM" src="https://user-images.githubusercontent.com/25887253/225988036-7fd847b8-d9c4-4ed8-be4f-f6dcd5348354.png">
<img width="1440" alt="Screenshot 2023-03-17 at 9 22 10 PM" src="https://user-images.githubusercontent.com/25887253/225988042-4410ad1d-6cef-4e62-9abd-be82f3f06a74.png">